package.json

```diff
@@ -82,6 +82,7 @@
     "vite": "^4.3.9",
     "vite-plugin-dts": "2.3.0",
     "vite-plugin-eslint": "^1.8.1",
+    "vite-plugin-static-copy": "^1.0.1",
```

Comment:
Can you explain why we can't use `public` directory? https://vitejs.dev/guide/assets.html#the-public-directory

Reply:
I tried using the public directory, but the assets weren't loading for some reason. However, if I copy it outside the public directory, into our own `assets` directory, it works.

Also this copy plugin is to do what nx does with the `assets` config in `project.json` for the old webpack version, i.e. copy the assets from one spot to another when building/serving. But nx doesn't support the assets config for vite build for whatever reason

Reply:
Can you put something in the README about how to configure and add new assets in the README?